afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/ImportedCaseBuilder.java

```diff
@@ -150,8 +153,12 @@ public ImportedCase build() {
             throw new UncheckedIOException(e);
         }
         context.getStorage().setConsistent(info.getId());
+
         context.getStorage().flush();
 
+        context.getStorage().getEventsBus().pushEvent(new CaseImported(info.getId(),
+                context.getFolderInfo().getId(), file), NodeEventType.CASE_IMPORTED.name());
```

Comment:
File could be null here as it is just one way to import a case (mostly come from datasource directly). AFS path would be more relevant: 
```java
ImportedCase ic = ...
String path = ic.getPath().toString()
```